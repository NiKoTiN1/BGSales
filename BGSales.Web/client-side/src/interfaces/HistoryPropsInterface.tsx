export default interface HistoryPropsInterface {
  push(url: string): void;
  goBack: any;
}
