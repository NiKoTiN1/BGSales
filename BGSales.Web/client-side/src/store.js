import { createStore, combineReducers, applyMiddleware } from "redux";
import reducer from "./reducers";
import thunk from "redux-thunk";
var rootReducer = combineReducers({
    reducer: reducer,
});
var store = createStore(rootReducer, applyMiddleware(thunk));
export default store;
