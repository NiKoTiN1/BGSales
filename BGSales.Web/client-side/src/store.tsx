import { createStore, combineReducers, applyMiddleware } from "redux";
import reducer from "./reducers";
import thunk from "redux-thunk";

const rootReducer = combineReducers({
  reducer,
});

const store = createStore(rootReducer, applyMiddleware(thunk));

export default store;
