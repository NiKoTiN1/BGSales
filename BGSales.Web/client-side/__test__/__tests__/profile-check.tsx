import { Provider } from "react-redux";
import renderer from 'react-test-renderer';
import AdvertiserPersonProfile from "../../src/components/advertiser-person-profile";
import AdvertiserPersonProfileEdit from "../../src/components/advertiser-person-profile-edit";
import AppFooter from "../../src/components/app-footer";
import store from "../../src/store"
import React from 'react';
import HistoryPropsInterface from "../../src/interfaces/HistoryPropsInterface";

jest.mock("history");
jest.useFakeTimers()
describe("BGSales Profile module", () => {
  const history:any = 0;
 
  it("check be truth Advertiser person profile edit", () => {
    
    const wrapper = renderer.create(<Provider store={store}><AdvertiserPersonProfileEdit history={history}  /> </Provider>);
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
  it("check be truth Advertiser person profile", () => {
    const wrapper = renderer.create(<Provider store={store}><AdvertiserPersonProfile id={""} /> </Provider>);
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
