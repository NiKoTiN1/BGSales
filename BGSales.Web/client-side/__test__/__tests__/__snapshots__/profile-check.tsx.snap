// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BGSales Profile module check be truth Advertiser person profile 1`] = `
Array [
  <div
    className="container-error"
  >
    <img
      src={
        Object {
          "process": [Function],
        }
      }
    />
    <p>
      Error this page is not available
    </p>
  </div>,
  " ",
]
`;

exports[`BGSales Profile module check be truth Advertiser person profile edit 1`] = `
Array [
  <div
    className="container-error"
  >
    <img
      src={
        Object {
          "process": [Function],
        }
      }
    />
    <p>
      Error this page is not available
    </p>
  </div>,
  " ",
]
`;
